src/features/pastelPaperWalletGenerator/PastelPaperWalletModal.module.css

```diff
@@ -34,3 +34,32 @@
   cursor: pointer;
   outline: none;
 }
+
+.btn_download {
+  position: absolute;
+  top: 12px;
+  right: 94px;
+  display: block;
+  width: 36px;
+  height: 36px;
+  border-radius: 50%;
+  border: none;
+  background-color: transparent;
+}
+
+.btn_download:hover {
+  background-color: rgba(255, 255, 255, 0.1);
```

Comment:
For now it is okay, but mind that each color will be taken from `theme/colors` soon - we should not write any colors on the fly, but keep all the colors in one place instead

Reply:
Ok @adrianobetkal